src/stories/collapse.stories.mdx

```diff
@@ -87,4 +87,25 @@ Toggles visibility of content
   </Story>
 </Canvas>
 
+<Canvas>
+  <Story
+    name='No Background and No Border'
+    args={{
```

Comment:
Forgot to tell you that you need to add `variant: 'secretUI'` to the args

Reply:
I believe we weren't seeing the border as your browser is in dark mode.

Reply:
added